### PR TITLE
Part 1

### PR DESCRIPTION
**Pull request by pavelloz**

### CODE REVIEW

**Comment by pavelloz** [comment]
Part 2 will contain:

* tags
* filters

File: marketplace_builder/views/pages/api-reference/liquid/flow-control.liquid [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+The condition of an if, elsif or unless tag should be either a normal Liquid expression or a comparison using Liquid expressions.
+Note that the comparison operators are implemented by the "if"-like tags; they don't work anywhere else in Liquid.
+
+## Operators
+
+The available comparison operators are:
+
+* `==`, `!=`, `and` `<>` — equality and inequality (the latter two are synonyms)
+* `<`, `<=`, `>`, `>=` — less/greater-than
+* There's a secret special value `empty` (unquoted) that you can compare arrays to; the comparison is `true` if the array has no members.
```
Comment (Slashek) [inline marketplace_builder/views/pages/api-reference/liquid/flow-control.liquid]: There's also `blank` - more commonly used

File: marketplace_builder/views/pages/api-reference/liquid/flow-control.liquid [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+* `and`
+* `or`
+
+You can use the and and or operators to include more than one condition in a control flow tag. and and or can be chained together to create complex conditionals.
+
+If you use multiple and or or operators, note that and operators will be evaluated first, then or operators. You cannot use parentheses to simulate an order of operations and control the order of operator evaluation. Parentheses are invalid characters within Liquid tags and prevent your tags from working.
+
+### and
+
+The and operator lets you add additional conditions to a tag. A condition with an and will only be true if both the left and the right side of the condition are true.
```
Comment (Slashek) [inline marketplace_builder/views/pages/api-reference/liquid/flow-control.liquid]: We might want to explicitly state that the very common operators `&&` and `||` will NOT work but will not raise errors

File: marketplace_builder/views/pages/api-reference/liquid/introduction.liquid [hunk 3]
```diff
@@ -0,0 +1,96 @@
+---
+converter: markdown
+slug: api-reference/liquid/introduction
+searchable: true
+metadata:
+  title: API Reference - Liquid Introduction
+---
+
+**Liquid** is a template language used in platformOS to build dynamic pages, and to provide dynamic configuration (e.g. based on currently logged in user). Use Liquid to provide an authorization policy for forms and pages, or to specify notifications (email, SMS, API call). If you are not familiar with Liquid. We have added a lot of filters and tags to make your life easier.
```
Comment (diana-lakatos) [inline marketplace_builder/views/pages/api-reference/liquid/introduction.liquid]: "If you are not familiar with Liquid." sentence is unfinished.

**Review by diana-lakatos — APPROVED** [review]
Reviewed and edited. Thank you!

**Pull request merged**